src/components/Search/Search.tsx

```diff
@@ -73,7 +73,7 @@ export const Search = ({
       />
       <Button type="submit">
         <span className={isSmall ? 'usa-sr-only' : 'usa-search__submit-text'}>
-          Search
```

Comment:
So while this works and may be perfectly acceptable, I can see a use case where someone may want to the Button text to be different from the label. 

There's also the thought of following convention for localizations which we established in #990. There we added an `i18n` object prop which contained all the localization values. Since we only have one value to worry about here, that might feel like a bit much.

_My_ overall preference with this is to follow convention, even if it feels like overkill, in the name of uniform codebase. Plus, to handle the scenario of wanting label and button text to be different we would have to introduce a new prop anyways, so that would be half way to implementing our established i18n pattern.